src/utilities/filetypes/EpwFile.cpp

```diff
@@ -3887,6 +3887,17 @@ bool EpwFile::parse(std::istream& ifs, bool storeData) {
     return false;
   }
 
+  struct epw_string
```

Comment:
Since this is a type, wouldn't it be more aligned with our conventions to call it `EPWString`, or otherwise start with an uppercase letter?

Reply:
Changed `epw_string` to `EPWString`.